napari/layers/labels/labels.py

```diff
@@ -287,22 +288,20 @@ def __init__(
         affine=None,
         blending='translucent',
         cache=True,
-        color=None,
+        colormap=None,
         depiction='volume',
         experimental_clipping_planes=None,
         features=None,
         metadata=None,
         multiscale=None,
         name=None,
-        num_colors=49,
         opacity=0.7,
         plane=None,
         properties=None,
         projection_mode='none',
         rendering='iso_categorical',
         rotate=None,
         scale=None,
-        seed_rng=None,
```

Comment:
is this supposed to be gone like this with no warning?

Reply:
Yes. As it was introduced after 0.4.18 release.